var/spack/repos/builtin/packages/sphexa/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Sphexa(CMakePackage, CudaPackage, ROCmPackage):
+    """SPH and N-body simulation framework"""
+
+    homepage = "https://github.com/sphexa-org/sphexa"
+    url = "https://github.com/sphexa-org/sphexa/archive/v0.0.tar.gz"
```

Comment:
Optional, I just like it for consistency since the actual versions also have three components:
```suggestion
    url = "https://github.com/sphexa-org/sphexa/archive/v0.0.0.tar.gz"
```

Reply:
done